Testscripts/Windows/PERF-XDP-RX-DROP.ps1

```diff
@@ -92,7 +82,7 @@ collect_VM_properties
         # Terminate process if ran more than 5 mins
         # TODO: Check max installation time for other distros when added
         $timer = 0
-        while ((Get-Job -Id $testJob).State -eq "Running") {
+        while ($testJob -and ((Get-Job -Id $testJob).State -eq "Running")) {
```

Comment:
Why condition is enhanced. Are we fixing any other problem apart from clean up ?

Reply:
No. We are just making sure that testJob is not empty.